mesheryctl/internal/cli/root/connections/list.go

```diff
@@ -70,27 +70,25 @@ mesheryctl exp connections list --page 2
 		if cmd.Flags().Changed("page") {
 			url = fmt.Sprintf("%s/api/integrations/connections?page=%d", baseUrl, pageNumberFlag)
 		} else {
-			url = fmt.Sprintf("%s/api/integrations/connections?pagesize=all", baseUrl)
+			url = fmt.Sprintf("%s/api/integrations/connections", baseUrl)
```

Comment:
Is the API call correct for this one @singh1203 ?

Reply:
Apologies for wrong changes here.